indexer/config/config.go

```diff
@@ -100,7 +100,7 @@ type ChainConfig struct {
 	L1Contracts L1Contracts `toml:"l1-contracts"`
 	L2Contracts L2Contracts `toml:"-"`
 
-	// Bedrock starting heights only applicable for OP-Mainnet & OP-Goerli
+	// Bedrock starting heights only applicable for OP-Mainnet & OP-Sepolia
```

Comment:
I do not believe this config is necessary for sepolia, only op mainnet or other chains that migrated to being bedrock compatible

Reply:
Oh.. I understands. Thanks @tynes